README.md

```diff
@@ -109,13 +115,24 @@ You are now ready to clone the repository and run `npm install`.
 
 ##### Manual Development Environment Setup
 
-1. Install [Windows Subsystem for Linux (WSL)] on your machine.
-2. Install [Scoop] via `iwr -useb get.scoop.sh | iex`
-3. Install git, go, nvm, and unzip via `scoop install git go nvm unzip`
-4. Install NodeJS via `nvm install 14.17.0`
-  * Remember to use it by running `nvm use 14.17.0`
+1. Install [Windows Subsystem for Linux (WSL)] on your machine. Skip this step, if WSL is already installed.
+2. Open a privileged PowerShell prompt (hit Windows Key + `X` and open `Windows PowerShell (Admin)`).
+3. Install [Scoop] via `iwr -useb get.scoop.sh | iex`.
```

Comment:
Scoop doesn't need to be privileged (and probably shouldn't be).  Not sure what we need it for (only thing should be nvm, but I thought that prompted itself).

Reply:
`iwr` command doesn't work within normal `cmd` terminal (cmd.exe), so power shell is required to run the command `iwr -useb get.scoop.sh | iex`

Reply:
Right, but that doesn't need to be an Admin PowerShell window. (just a normal, non-Admin one).